File: app/src/main/java/com/hnam/exerciseweek3/MainActivity.java
package com.hnam.exerciseweek3;

import android.content.Intent;
import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.support.v7.widget.RecyclerView;
import android.view.View;
import android.widget.Button;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        Button btnListView = findViewById(R.id.btn_listView);
        btnListView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(MainActivity.this, ListViewActivity.class);
                startActivity(i);
            }
        });

        Button btnRecyclerView = findViewById(R.id.btn_recyclerView);
        btnRecyclerView.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                Intent i = new Intent(MainActivity.this, RecyclerViewActivity.class);
                startActivity(i);
            }
        });
    }
}

File: app/src/main/java/com/hnam/exerciseweek3/MainActivity.kt
package com.hnam.exerciseweek3

import android.content.Intent
import android.os.Bundle
import android.widget.Button
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        val btnListView = findViewById<Button>(R.id.btn_listView)
        btnListView.setOnClickListener {
            val i = Intent(this@MainActivity, ListViewActivity::class.java)
            startActivity(i)
        }

        val btnRecyclerView = findViewById<Button>(R.id.btn_recyclerView)
        btnRecyclerView.setOnClickListener {
            val i = Intent(this@MainActivity, RecyclerViewActivity::class.java)
            startActivity(i)
        }
    }
}
